public/bitemp_graph_helper.js

```diff
@@ -228,15 +246,15 @@ function saveNewDoc() {
   var docData;
 
   if (format === 'JSON') {
+    docData = jQuery.parseJSON(data);
   } else {
     data = data.replace(/ /g, '');
   }
   $.ajax({
-    url: '/v1/documents/?temporal-collection=' + selectedColl,
+    url: '/v1/documents?uri=' + newURI,
```

Comment:
shouldn't this be specifying the temporal collection? Otherwise, how will it be a temporal doc?